pkg/controller/gkenetworkparamset/gkenetworkparamset_controller_test.go

```diff
@@ -1013,37 +1009,6 @@ func TestHandleGKENetworkParamSetDelete_NetworkInUse(t *testing.T) {
 				}
 			},
 		},
-		{name: "Network switches params while InUse",
```

Comment:
Confirmed with @sdmodi, these test cases aren't actually possible. Network is immutable after creation.